package.json

```diff
@@ -18,7 +18,7 @@
     "build:umd": "cross-env BABEL_ENV=commonjs NODE_ENV=development webpack src/index.js dist/react-redux-firebase.js",
     "build:umd:min": "cross-env BABEL_ENV=commonjs NODE_ENV=production webpack src/index.js dist/react-redux-firebase.min.js",
     "build": "npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min",
-    "watch": "npm run build:commonjs -- --watch",
+    "watch": "npm run build:umd -- --watch",
```

Comment:
Nice. This is the more often use-case since most are requiring and using Webpack or some other bundler that points to the UMD version.